installscripts/dockerfiles/jenkins-ce/Dockerfile

```diff
@@ -2,12 +2,18 @@
 FROM jenkins/jenkins:2.121.3
 MAINTAINER JazzOSS Team
 
+ARG goVersion=1.10.3
 # Switching to root to configure the image with system packages
 USER root
 RUN apt-get update && apt-get install -y vim curl sudo libtool autoconf make unzip rsync gcc autogen shtool pkg-config lsb-release python python-dev python-pip python-setuptools groff less && \
 curl https://bootstrap.pypa.io/get-pip.py | python && pip install --upgrade awscli && apt-get clean && pip install virtualenv && /usr/bin/easy_install virtualenv
 RUN wget -O /opt/apache-maven-3.5.2-bin.tar.gz https://archive.apache.org/dist/maven/maven-3/3.5.2/binaries/apache-maven-3.5.2-bin.tar.gz && tar xzvf /opt/apache-maven-3.5.2-bin.tar.gz -C  /opt && export PATH=$PATH:/opt/apache-maven-3.5.2/bin >> /etc/profile.d/maven.sh && ln -sf /opt/apache-maven-3.5.2/bin/mvn /usr/bin/mvn
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash && apt-get install -y nodejs && npm install  -global serverless@1.30.0 @angular/cli@1.7.3 jshint
+# Install scripts for golang 
+RUN curl -O https://storage.googleapis.com/golang/go${goVersion}.linux-amd64.tar.gz && tar -xvf go${goVersion}.linux-amd64.tar.gz && mv go /usr/local && \
+echo 'export GOROOT=/usr/local/go' | sudo tee -a /etc/profile  && echo 'export GOBIN=$GOROOT/bin' | sudo tee -a /etc/profile &&  echo 'export GOPATH=$GOROOT/src' | sudo tee -a /etc/profile && \
+echo 'export PATH=$PATH:$GOPATH:$GOBIN' | sudo tee -a /etc/profile && source /etc/profile && RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
@Johnbush Please remove the extra RUN added here.

Reply:
ok